src/components/views/MoviesSummary.vue

```diff
@@ -0,0 +1,109 @@
+<script>
+import { defineComponent } from 'vue';
+import TheTag from '../common/TheTag.vue';
```

Comment:
A small tip.
Currently our vite configuration gives us an alias to easily import things whenever we need like so:

```suggestion
import TheTag from '@/components/common/TheTag.vue';
```

I'd suggest to try to use the alias as much as possible and as such let us be more consistent 💡

Reply:
Will apply! 👍